chandra_models/xija/pftank2t/pftank2t_spec.json

```diff
@@ -145,7 +145,17 @@
             1110
         ]
     },
-    "limits": {},
+    "limits": {
+        "pftank2t": {
+            "odb.caution.high": 120,
+            "odb.caution.high.matlab_identifier": "yellow_limit",
+            "odb.warning.high": 140,
+            "odb.warning.high.matlab_identifier": "red_limit",
+            "planning.warning.high": 110,
```

Comment:
This limit was just updated in the MATLAB dev branch (Trunk) to be `115` I'm not sure when this change was expected to go live, but it should probably be updated here too.